### ISSUE TITLE
adversarial training 

### ISSUE THREAD

**Issue by soans1994** [MoleImg/Domain_Adversarial_Net#1]
hello author,

can you update code to train the model using adversarial loss.
how can i mutiply loss of discriminator as feedback to segmentation loss like in pytorch in keras.
